schnorr_fun/src/frost.rs

```diff
@@ -3,17 +3,17 @@
 //! ## Synopsis
 //!
 //! ```
+//! # use schnorr_fun::binonce::NonceKeyPair;
 //! use schnorr_fun::{
-//!     binonce::NonceKeyPair,
 //!     frost,
 //!     Message,
 //! };
 //! use rand_chacha::ChaCha20Rng;
 //! use sha2::Sha256;
 //! // use sha256 to produce deterministic nonces -- be careful!
-//! let proto = frost::new_with_deterministic_nonces::<Sha256>();
+//! let frost = frost::new_with_deterministic_nonces::<Sha256>();
```

Comment:
nice I prefer this as `frost` over `proto`

Reply:
I had to change it for some reason but it worked when I changed it back!